apps/app-article-server/src/browser/index.html

```diff
@@ -30,38 +30,95 @@
     <!--Start Google Tag Manager-->
     <script>
       (function (w, d, s, l, i) {
-	w[l] = w[l] || [];
-	w[l].push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
-	var f = d.getElementsByTagName(s)[0],
-	  j = d.createElement(s),
-	  dl = l != "dataLayer" ? "&l=" + l : "";
-	j.async = true;
-	j.src = "//www.googletagmanager.com/gtm.js?id=" + i + dl;
-	f.parentNode.insertBefore(j, f);
+        w[l] = w[l] || [];
+        w[l].push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != "dataLayer" ? "&l=" + l : "";
+        j.async = true;
+        j.src = "//www.googletagmanager.com/gtm.js?id=" + i + dl;
+        f.parentNode.insertBefore(j, f);
       })(window, document, "script", "dataLayer", "GTM-MH7NXT3");
     </script>
     <!--End Google Tag Manager-->
     <!--Start Google Ad-->
-    <script async src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
+    <script src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
     <script>
       var ksf = {};
       ksf.settings = {};
       ksf.settings.brand = {};
       ksf.settings.brand.id = "hbl";
     </script>
-    <script async src="https://script.ksfmedia.fi/app/gamAppAds.min.js"></script>
+
+    <script src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
+    <script>
+      const getUrlParam = () => {
+        return new URLSearchParams(window.location.search);
+      };
+    
+      const getBrandValueParam = () => {
+        let urlParams = getUrlParam();
+        if (urlParams.has("paper")) {
+          return urlParams.get("paper");
+        }
+        return "hbl";
+      };
+
+      const brandValueName = getBrandValueParam()
+
+      if (brandValueName == "on") {
+        appToTarget = "appON";
```

Comment:
Seems that this variable not defined anywhere (or is it?). We should probably do it.